src/components/fruitCounterList/fruitCounterList.jsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from "react";
+import FruitCounter from "../fruitCounter/fruitCounter.jsx";
+import Button from "../button/button.jsx";
+
+function FruitCounterList() {
+    const [fruitCounters, setFruitCounters] = useState({
+        Aardbeien: 0,
+        Bananen: 0,
+        Appels: 0,
+        "Kiwi's": 0
+    });
+
+    const updateCount = (fruit, amount) => {
+        setFruitCounters((prev) => ({
+            ...prev,
+            [fruit]: Math.max(0, prev[fruit] + amount)
+        }));
+    };
+
+    const reset = () => {
+        setFruitCounters({
+            Aardbeien: 0,
+            Bananen: 0,
+            Appels: 0,
+            "Kiwi's": 0
+        });
+    };
+
+    return (
```

Comment:
Zelf zou ik dit gewoon in de app.jsx zetten. want nu staat het als component maar het is niet echt een component